README.md

```diff
@@ -1,4 +1,4 @@
-[![github repo badge](https://img.shields.io/badge/github-repo-000.svg?logo=github&labelColor=gray&color=blue)](https://github.com/MiBiPreT/anatrans)[![PyPI version](https://badge.fury.io/py/anatrans.svg)](https://badge.fury.io/py/anatrans)[![github license badge](https://img.shields.io/github/license/MiBiPreT/anatrans)](https://github.com/MiBiPreT/anatrans) [![RSD](https://img.shields.io/badge/rsd-anatrans-00a3e3.svg)](https://www.research-software.nl/software/anatrans) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10877886.svg)](https://doi.org/10.5281/zenodo.10877886) [![workflow cii badge](https://bestpractices.coreinfrastructure.org/projects/8710/badge)](https://bestpractices.coreinfrastructure.org/projects/8710) [![fair-software.eu](https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F-green)](https://fair-software.eu) [![workflow scq badge](https://sonarcloud.io/api/project_badges/measure?project=MiBiPreT_anatrans&metric=alert_status)](https://sonarcloud.io/dashboard?id=MiBiPreT_anatrans) [![workflow scc badge](https://sonarcloud.io/api/project_badges/measure?project=MiBiPreT_anatrans&metric=coverage)](https://sonarcloud.io/dashboard?id=MiBiPreT_anatrans) [![documentation](https://github.com/MiBiPreT/anatrans/actions/workflows/documentation.yml/badge.svg)](https://mibipret.github.io/anatrans) [![build](https://github.com/MiBiPreT/anatrans/actions/workflows/build.yml/badge.svg)](https://github.com/MiBiPreT/anatrans/actions/workflows/build.yml) [![cffconvert](https://github.com/MiBiPreT/anatrans/actions/workflows/cffconvert.yml/badge.svg)](https://github.com/MiBiPreT/anatrans/actions/workflows/cffconvert.yml) [![sonarcloud](https://github.com/MiBiPreT/anatrans/actions/workflows/sonarcloud.yml/badge.svg)](https://github.com/MiBiPreT/anatrans/actions/workflows/sonarcloud.yml) [![markdown-link-check](https://github.com/MiBiPreT/anatrans/actions/workflows/markdown-link-check.yml/badge.svg)](https://github.com/MiBiPreT/anatrans/actions/workflows/markdown-link-check.yml)
```

Comment:
Badges are beautiful. The documentation badge doesn't work here but it does in the non-diff-view of the file: https://github.com/MiBiPreT/anatrans/tree/fix_docs_badge

Looks good!